docs/kubeedge.md

```diff
@@ -0,0 +1,70 @@
+# What is KubeEdge
+
+## The KubeEdge mission
+
+Our goal is to make an open platform to enable Edge computing, extending native containerized
+application orchestration capabilities to hosts at Edge. which built upon kubernetes and provides
+fundamental infrastructure support for network, app.deployment and metadata synchronization
+between cloud and edge. It also supports `MQTT` and allows developers to author customer logic
+and enable resource constraint devices communication at Edge. Kubeedge is *the open platform to enable Edge computing*.
+The advantages of Kubeedge include mainly:
+
+* **Edge Computing**

```

Comment:
please see the readme.md in kubeedge/kubeedge. The description has been revised.

Reply:
Fixed this. Referred the updated content and made changes accordingly.